examples/preprocessing/plot_css.py

```diff
@@ -0,0 +1,96 @@
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Load sample subject data
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+ave_fname = data_path + '/MEG/sample/sample_audvis-no-filter-ave.fif'
+cov_fname = data_path + '/MEG/sample/sample_audvis-cov.fif'
+trans_fname = data_path + '/MEG/sample/sample_audvis_raw-trans.fif'
+bem_fname = subjects_dir + '/sample' + '/bem' + '/sample-5120-bem-sol.fif'
+
+raw = mne.io.read_raw_fif(data_path + '/MEG/sample/sample_audvis_raw.fif')
+fwd = mne.read_forward_solution(fwd_fname)
+fwd = mne.convert_forward_solution(fwd, force_fixed=True, surf_ori=True)
+fwd = mne.pick_types_forward(fwd, meg=True, eeg=True, exclude=raw.info['bads'])
+cov = mne.read_cov(cov_fname)
+
+###############################################################################
+# Find patches (labels) to activate
+all_labels = mne.read_labels_from_annot(subject='sample',
+                                        subjects_dir=subjects_dir)
+labels = []
+for select_label in ['parahippocampal-lh', 'postcentral-rh']:
+    labels.append([lab for lab in all_labels if lab.name in select_label][0])
+hiplab, postcenlab = labels
+
+###############################################################################
+# Simulate one cortical dipole (40 Hz) and one subcortical (239 Hz)
+
+
+def cortical_waveform(times):
+    return 10e-9 * np.cos(times * 2 * np.pi * 40)
+
+
+def subcortical_waveform(times):
+    return 10e-9 * np.cos(times * 2 * np.pi * 239)
+
+
+times = np.linspace(0, 0.5, int(0.5 * raw.info['sfreq']))
+stc = simulate_sparse_stc(fwd['src'], n_dipoles=2, times=times,
+                          location='center', subjects_dir=subjects_dir,
+                          labels=[postcenlab, hiplab],
+                          data_fun=cortical_waveform)
+stc.data[np.where(np.isin(stc.vertices[0], hiplab.vertices))[0], :] = \
+    subcortical_waveform(times)
+evoked = simulate_evoked(fwd, stc, raw.info, cov, nave=15)
+
+###############################################################################
+# Process with CSS and plot PSD of EEG data before and after processing
+evoked_subcortical = mne.preprocessing.cortical_signal_suppression(evoked,
+                                                                   n_proj=6)
+chs = mne.pick_types(evoked.info, meg=False, eeg=True)
+pss = np.mean(np.array([plt.psd(evoked.data[x, :], Fs=evoked.info['sfreq'])
+                        for x in chs]), axis=0)[0]
+pss_proc = np.mean(np.array([plt.psd(evoked_subcortical.data[x, :],
+                                     Fs=evoked_subcortical.info['sfreq'])
+                            for x in chs]), axis=0)[0]
+fr = plt.psd(evoked.data[mne.pick_types(evoked.info, meg='mag'), :][0, :],
+             Fs=evoked.info['sfreq'])[1]
+plt.close('all')
```

Comment:
for our docs to build properly, we don't need to explicitly `close()` or `show()` figures.  sphinx-gallery will handle that for us.